src/package.h

```diff
@@ -11,10 +11,18 @@ namespace cyclus {
 
 /// Packager is a class that packages materials into discrete items in ways that mimic realistic nuclear material handling. Packages will eventually be a required parameter of resources.
 class Package {
-
+  friend class Context;
   public:
     typedef boost::shared_ptr<Package> Ptr;
 
+    // create a new package type with default values. Should be called by 
+    // the context only (see Context::AddPackage), unless you want an 
+    // untracked package type (which you probably don't)
+    static Ptr Create();
```

Comment:
You don't appear to call this anywhere anymore - still needed?

Reply:
See above. Maybe since it's not currently used, it can be safely removed and only re-introduced if the need arises